src/formatron/integrations/transformers.py

```diff
@@ -12,23 +12,24 @@
 from formatron.integrations._utils import get_original_characters
 
 
-def create_engine_vocabulary(tokenizer: PreTrainedTokenizerBase) -> kbnf.Vocabulary:
+def create_engine_vocabulary(tokenizer: PreTrainedTokenizerBase, vocab_processors: typing.Optional[list[typing.Callable]] = None) -> kbnf.Vocabulary:
```

Comment:
These new parameters also look good and I think we can now extend exllamav2 and vllm's integration in the same way, which should be essentially the same boilerplate. Also, I think it's better to add some explicit documentation for vocab_processors, since its name is not as intuitive as other parameters.